tests/api_tests.py

```diff
@@ -9,36 +9,36 @@
 from populate_arena import populate_arena
 from clean_arena    import clean_arena
 
-import ev3dev as ev3
```

Comment:
Why not just change this line to `import ev3dev2 as ev3`?

Reply:
Because it's no longer generic "ev3" interface... It's just the couple of utility methods we dumped into the init file. Do you have a different name suggestion?

Reply:
Ok, that is a good point. But `ev3` is still shorter than `ev3dev2`, and `ev3dev2` still contains `ev3` in the name. So it is kind of not generic in the first place.

Reply:
Agreed :laughing: I'll have to think about it a bit and see what I think in the morning (this 12-hour offset makes communication really weird!)